src/test/java/ttt/GameTest.java

```diff
@@ -0,0 +1,95 @@
+package ttt;
+
+import org.junit.Test;
+
+import java.io.StringReader;
+import java.io.StringWriter;
+
+import static org.hamcrest.Matchers.is;
+import static org.junit.Assert.assertThat;
+import static ttt.PlayerSymbol.*;
+
+public class GameTest {
+
+    @Test
+    public void playersTakeTurnsUntilTheBoardIsFull() {
+        PlayerSpy player1 = new PlayerSpy(createCommandPromptToReadInput("0\n4\n5\n6\n7\n"), X);
+        PlayerSpy player2 = new PlayerSpy(createCommandPromptToReadInput("1\n2\n3\n8\n"), O);
+
+        Game game = new Game(new Board(),
+                createCommandPromptToReadInput(""),
+                player1,
+                player2);
+
+        game.play();
+
+        assertThat(player1.numberOfTurnsTaken(), is(5));
+        assertThat(player2.numberOfTurnsTaken(), is(4));
+    }
+
+    @Test
+    public void displaysCongratulatoryMessageWhenThereIsAWinningFormation() {
+        PromptSpy gamePrompt = new PromptSpy();
+
+        Game game = new Game(new Board(),
+                gamePrompt,
+                createHumanPlayer(createCommandPromptToReadInput("0\n1\n2\n"), X),
+                createHumanPlayer(createCommandPromptToReadInput("4\n7\n"), O));
+
+        game.play();
+
+        assertThat(gamePrompt.getLastMessagePrinted(), is("Congratulatory message printed"));
```

Comment:
I like this idea, as it makes the class a proper spy, rather than a mix between a spy and a stub returning canned answers.